.github/workflows/test.yml

```diff
@@ -0,0 +1,52 @@
+name: test
+on: [push, pull_request]
+permissions:
+  contents: read
+
+jobs:
+  test:
+    name: Ruby ${{ matrix.ruby }}, ${{ matrix.gemfile }}.gemfile
+    runs-on: ubuntu-latest
+    services:
+      test-postgres:
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_PASSWORD: foobar123
+    strategy:
+      matrix:
+        gemfile: ['rails_7.1', 'rails_7.2']
```

Comment:
soon I'd want to test with rails 8.0 as well, but depends on `pg_party` supporting rails 8 before that can happen.
Likely to be resolved soon as per https://github.com/rkrage/pg_party/pull/86